ApplyToBecomeInternal/ApplyToBecomeInternal/wwwroot/package-lock.json

```diff
@@ -1,6 +1,2180 @@
 {
```

Comment:
I'd recommend undoing this change for the moment - but we should probably as a team define what version of node we want to use 

```bash
git checkout main -- ApplyToBecomeInternal/ApplyToBecomeInternal/wwwroot/package-lock.json
``` 

should reset it